README.md

```diff
@@ -13,21 +13,34 @@ The Open Grid Emissions Dataset can be [downloaded here](https://singularity.ene
 
 To install and run the pipeline on your computer, open anaconda prompt, navigate to the folder where you want to save the repository, and run the following commands:
 
-```
+```bash
 conda install git
 git clone https://github.com/singularity-energy/open-grid-emissions.git
 conda update conda
 cd open-grid-emissions
 conda env create -f environment.yml
 conda activate open_grid_emissions
+```
+A more detailed walkthough of these steps can be found below in the "Development Setup" section.
+
+It is also possible to use `pipenv`:
+```bash
+pipenv sync
+pipenv shell
+```
+since the dependencies will be installed in an isolated environment. It is of course possible to install the dependencies using the requirements file:
+```bash
+pip install -r requirements.txt
```

Comment:
Is this something that needs to be done in addition to the pipenv commands above, or is this an alternative setup? We should make this clear.

Reply:
I will need clarify

Reply:
I have decided to not mention this installation method. Users who want to install from the requirements will create a virtual environment with the tool of their choice and install dependencies therein. I don't think it worth going through the details.